fsw/src/lc_msgdefs.h

```diff
@@ -161,7 +161,7 @@
  *       Successful execution of this command may be verified with
  *       the following telemetry:
  *       - #LC_HkPacket_t.CmdCount will be cleared
- *       - The #LC_RESET_DBG_EID debug event message will be
+ *       - The #LC_RESET_INF_EID debug event message will be
```

Comment:
if LC_RESET_INF_EID is to be changed to of an INFORMATION type, these comments may need updating

```suggestion
 *       - The #LC_RESET_INF_EID informational event message will be
```